pager/src/main/java/com/google/accompanist/pager/Pager.kt

```diff
@@ -81,6 +81,37 @@ private data class PageData(val page: Int) : ParentDataModifier {
 private val Measurable.page: Int
     get() = (parentData as? PageData)?.page ?: error("No PageData for measurable $this")
 
+/**
+ * Contains the default values used by [HorizontalPager] and [VerticalPager].
+ */
+@ExperimentalPagerApi
+object PagerDefaults {
+    /**
+     * Create and remember default [FlingBehavior] that will represent the scroll curve.
+     *
+     * @param state The [PagerState] to update.
+     * @param decayAnimationSpec The decay animation spec to use for decayed flings.
+     * @param snapAnimationSpec The animation spec to use when snapping.
+     */
+    @Composable
+    fun defaultPagerFlingConfig(
+        state: PagerState,
+        decayAnimationSpec: DecayAnimationSpec<Float> = defaultDecayAnimationSpec(),
+        snapAnimationSpec: AnimationSpec<Float> = spring(stiffness = SnapSpringStiffness),
+    ): FlingBehavior = remember(state, decayAnimationSpec, snapAnimationSpec) {
+        object : FlingBehavior {
+            override suspend fun ScrollScope.performFling(
+                initialVelocity: Float
+            ): Float = state.fling(
```

Comment:
Thanks! That looks very flexible! 

I was wondering if we could just inline the state.fling here? This would prove that this flingBehaviour is creatable via public API of the PagerState only and people are able to create their own (similar or different ones)

Reply:
It's possibly, but I'd need to make some internal state (such as `pageSize`) public. Not sure if that's the right thing here, but I'll add an issue to revisit this.

Reply:
#257